src/themes/dark-theme/color-component-tokens.js

```diff
@@ -357,6 +357,8 @@ export default (
   inputEnhancerFill: themePrimitives.mono500,
   inputEnhancerFillDisabled: themePrimitives.mono700,
   inputEnhancerTextDisabled: themePrimitives.mono500,
+  inputPlaceholder: themePrimitives.primary400,
+  inputPlaceholderDisabled: themePrimitives.primary400,
```

Comment:
I think these should be `mono` colors since some folks will have a `primary` color that is blue or green or whatever.

Reply:
I changed these to use the `mono` colors